integration/script/run-test

```diff
@@ -161,22 +163,22 @@ const startTestDemo = () => {
 
 const waitUntilTestDemoStarts = async () => {
   console.log(green, ' Waiting for test demo to start');
-  count = 0;
-  threshold = 60;
-
-  while (count < 60) {
-    const isInUse = await checkIfPortIsInUse(8080);
-    if (isInUse === true) {
-      console.log(green, ' Test demo has started successfully');
-      return;
+    count = 0;
+    threshold = 60;
+  
+    while (count < threshold) {
+      const isInUse = await checkIfPortIsInUse(8080);
+      if (isInUse === true) {
+        console.log(green, ' Test demo has started successfully');
+        return;
+      }
+      count += 1;
+      await new Promise((resolve) => setTimeout(resolve, 1000));
     }
-    count += 1;
-    await new Promise((resolve) => setTimeout(resolve, 1000));
-  }
-
-  console.log(red, ' Test demo did not start successfully');
-  terminateTestDemo(pid);
-  exit(1);
+  
+    console.log(red, ' Test demo did not start successfully');
+    terminateTestDemo();
+    exit(1);
```

Comment:
Seems indent is not correct here.

Reply:
prettified the whole file and renamed to `run-test.js` from just `run-test`.